lib/osutil/sync_windows.go

```diff
@@ -0,0 +1,14 @@
+// +build windows
```

Comment:
Headers.

Yet perhaps it's easier just to share the implementation across all platforms, and do `runtime.GOOS == "windows"` in the sync dir case?